src/test/skript/README.md

```diff
@@ -94,3 +94,801 @@ Server files are located at <code>build/test_runners</code>.
 
 To run individual test files, use <code>/sk test \<file\></code>. To run last
 used file again, just use <code>/sk test</code>.
+
+## Test Coverage
```

Comment:
this is bound to become outdated, should we keep it?

Reply:
I'm ok with adding it if you, @TFSMads, or someone else is willing to take responsibility in occasionally checking to see if it's up to date (e.g. monthly, after every release). It's not something I have the time or will to ensure is up to date so I'd like someone who's able to keep track of it. If it's not updated, it won't be useful.

It's not currently up to date though so that needs fixing first, regardless.